services/src/main/scala/cromwell/pipeline/service/ProjectConfigurationService.scala

```diff
@@ -1,21 +1,19 @@
 package cromwell.pipeline.service
 
 import cromwell.pipeline.datastorage.dao.repository.ProjectConfigurationRepository
-import cromwell.pipeline.datastorage.dto.{
-  FileParameter,
-  ProjectConfiguration,
-  ProjectFileConfiguration,
-  ProjectId,
-  TypedValue
-}
+import cromwell.pipeline.datastorage.dto._
 import cromwell.pipeline.model.wrapper.UserId
 
 import java.nio.file.Path
 import scala.concurrent.{ ExecutionContext, Future }
 
 trait ProjectConfigurationService {
 
-  def addConfiguration(projectConfiguration: ProjectConfiguration, userId: UserId): Future[Unit]
+  def addConfiguration(
```

Comment:
I just realized that it's wrong to change signatures of `Service` methods when we change API scheme. This method should still accept whole `ProjectConfiguration`, now it's up to `Controller` layer to assemble it.

Reply:
I thout, that controller didn`t create new entities. 
But ok, I can change this.

Reply:
I should not contain businesses logic, but it is allowed to trivially map data from one form to another.
The fact that `projectId` comes from different place than other configuration info is purely caused by our changes of API scheme, no business logic involved. So now it's up to controller layer to make inputs match `Service` layer expectations.

Reply:
Done